Document HEOS browse feature (#38037)
home-assistant/home-assistant.github.io: source/_integrations/heos.markdown

- Controlling play mode (e.g., play/pause), volume, mute, and shuffle
- Playing HEOS favorites, playlists, quick selects, URLs
- Setting the source to physical inputs (e.g., `AUX1`)

> - Browsing HEOS music services (e.g., Tidal) and sources (.e.g., Favorites)

Reviewer: ```suggestion
- Browsing HEOS music services (for example, **Tidal**) and sources (such as **Favorites**)
```
the term "e.g." is on the no-no list :smile:  in our [style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg).
